selfdrive/test/test_onroad.py

```diff
@@ -72,7 +72,7 @@
 
 PROCS.update({
   "tici": {
-    "./pandad": 4.0,
```

Comment:
didn't we fix the thresholds so a diff this small shouldn't make the test fail?

Reply:
We did. This is the first failure in over 1000 runs that failed because of the new thresholds (https://jenkins.comma.life/blue/organizations/jenkins/openpilot/detail/__jenkins_loop_test_4/14/pipeline/#step-151-log-865)
We can make them even more tolerant, but for now they seem to hold up.

Reply:
4 vs 5 definitely shouldn't fail though

Reply:
oh it's 11% in the failed test run. maybe a real issue?